tests/performance/scripts/runperf.sh

```diff
@@ -37,8 +37,8 @@ cat >config.json<<END
         "node_num": 10,
         "imagerepo": "pavan187",
         "k8smaster2": "http://$MASTER_IP_2:12458",
-        "cloudimageurl": "pavan187/cloudcore:v1.0",
-        "edgeimageurl": "pavan187/edgecore:latest",
+        "cloudimageurl": "pavan187/cloudcore:v2.1",
```

Comment:
can we use kubeedge dockerhub registry ?
cc @m1093782566

Reply:
@sids-b Currently it is better to use the temp docker registry Since we will do lots of tests.
and Actually in the official image will not include some test codes like stubs in the test image.